docs/components/servo/pi.md

```diff
@@ -0,0 +1,99 @@
+---
+title: "Configure a pi Servo"
+linkTitle: "pi"
+weight: 90
+type: "docs"
+description: "Configure a pi servo."
+tags: ["servo", "components"]
+icon: "img/components/servo.png"
+# SME: Rand
+---
+
+{{% alert title="Note" color="note" %}}
+
+Unlike other servo models, `pi` servos are implemented as part of the [`pi` board component](https://github.com/viamrobotics/rdk/blob/main/components/board/pi/impl/servo.go).
+
+{{% /alert %}}
+
+Configure a `pi` servo to integrate a hobby servo controlled by general-purpose input/output (GPIO) pins on a [Raspberry Pi board](/components/board/pi) into your robot:
+
+{{< tabs name="Configure a pi Servo" >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and navigate to the **Create component** menu.
+Enter a name for your servo, select the type `servo`, and select the `pi` model.
+
+Click **Create component**:
+
+![An example configuration for a pi servo in the Viam app Config Builder.](../img/pi-servo-ui-config.png)
+
+Edit and fill in the `"attributes"` JSON to align with your `board` `name` and GPIO wiring.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "name": "<your-servo-name>",
+            "type": "servo",
+            "model": "pi",
+            "attributes": {
+                "pin": "<your-pin-number>",
+                "board": "<your-board-name>"
+            }
+        }, ... <insert your board component config>
+    }
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+        "name": "your-servo-name",
+        "type": "servo",
+        "model": "pi",
+        "attributes": {
+            "pin": "16",
+            "board": "your-board-name"
+        }
+    }, 
+    {
+        "name": "your-board-name",
+        "type": "board",
+        "model": "pi"
```

Comment:
```suggestion
    {
        "name": "local",
        "type": "board",
        "model": "pi"
    }, 
    {
        "name": "my_servo",
        "type": "servo",
        "model": "pi",
        "attributes": {
            "pin": "16",
            "board": "local"
        }
```

Reply:
@npentrel @JessamyT Same here as above-- Maybe board first and "my_board" instead of "local" could be a good in-between for this template?